### PR TITLE
Spaces

### PR DESCRIPTION
**Pull request by sashamelentyev**

### CODE REVIEW

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks! Wonder if it would worth it to add a tidy check for that...

**Comment by alexander-melentyev** [comment]
You are welcome! Why not?

**Comment by GuillaumeGomez** [comment]
For me we should, but not sure if it's worth it, that's pretty much all. :)

Considering you made changes to `.patch` files, I'll wait for someone from the @rust-lang/infra team to take a look.

**Comment by alexander-melentyev** [comment]
Ok

**Comment by calebcartwright** [comment]
Some of these changes are stripping two consecutive spaces off the end of lines in markdown files, which I believe is significant/possibly intentional due to how markdown does line breaking.

Are we sure all these changes are desired, including for the tools that are primarily developed externally?

**Comment by alexander-melentyev** [comment]
@calebcartwright, You're absolutely right about the two spaces. Add two spaces back

File: src/ci/docker/host-x86_64/disabled/riscv64gc-linux/0001-Remove-stime-function-calls.patch [hunk 1]
```diff
@@ -89,8 +89,8 @@ index 70f829e7f..878375d78 100644
  				bb_perror_msg_and_die("can't set time of day");
 +		}
  	}
- 
+
  	if (flags != 1) /* not lone -s */
--- 
+--
 2.25.1
```
Comment (bjorn3) [inline src/ci/docker/host-x86_64/disabled/riscv64gc-linux/0001-Remove-stime-function-calls.patch:96]: Patch files shouldn't be changed IMO. These spaces are inserted by git, so removing them would mean that applying and recreating the patch would result in a different patch file.

Comment (alexander-melentyev) [reply src/ci/docker/host-x86_64/disabled/riscv64gc-linux/0001-Remove-stime-function-calls.patch]: I delete changes from .patch files. Thank you!

**Comment by alexander-melentyev** [comment]
@bjorn3, need your review.

File: src/ci/docker/host-x86_64/dist-aarch64-linux/aarch64-linux-gnu.config [hunk 2]
```diff
@@ -352,23 +352,23 @@ CT_LIBC_GLIBC_USE_PORTS=y
 CT_LIBC_ADDONS_LIST=""
 
 #
-# WARNING !!!                                            
+# WARNING !!!
```
Comment (bjorn3) [inline src/ci/docker/host-x86_64/dist-aarch64-linux/aarch64-linux-gnu.config:355]: > ```
> # Automatically generated file; DO NOT EDIT.
> ```

Looks like this needs to be reverted as well.

**Review by bjorn3 — COMMENTED** [review]
After you have reverted the indicated change please squash all commits to keep the git history a bit cleaner and avoid polluting the `git blame` output.

**Comment by alexander-melentyev** [comment]
@bjorn3 thank you! All your recommended - done. Please check.

**Comment by bjorn3** [comment]
r? @bjorn3

> Considering you made changes to .patch files, I'll wait for someone from the `@rust-lang/infra` team to take a look.

All `.patch` file and other infra file changes are reverted.

@bors r+ rollup=always

**Pull request merged**